src/main/java/mate/academy/model/MovieSession.java

```diff
@@ -0,0 +1,62 @@
+package mate.academy.model;
+
+import java.time.LocalDateTime;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+
+@Entity
+public class MovieSession {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+    @OneToOne
+    private Movie movie;
```

Comment:
In my point of view, it's better to use @ManyToOne here. Why? The explanation is next. A lot of movie sessions may have the same movie. Let's imagine that you are an owner of a cinema and your cinema has three movie sessions with the same movie and the same cinema hall on a particular day. For instance, it may be Home Alone 2. So, you may have three movie sessions with three identical movies and cinema halls. The difference between them will only be the time of beginning them. The first one will start at 9 o'clock, the second one will start at 10 o'clock, the third one will start at 11 o'clock and so on.